pkg/skaffold/runner/dev.go

```diff
@@ -60,7 +63,8 @@ func (r *SkaffoldRunner) doDev(ctx context.Context, out io.Writer) error {
 	// if any action is going to be performed, reset the monitor's changed component tracker for debouncing
 	defer r.monitor.Reset()
 	defer r.listener.LogWatchToUser(out)
-
+	event.DevLoopInProgress(iteration)
```

Comment:
Should these not follow the var-for-testing approach used for fileSync*?

Reply:
That is a good question.  I also thought, i would have to add test variables. 
But in the same func below, the `event.ResetStateOnDeploy` isn't mocked out in tests. The unit tests pass so i did not add variables for these.